data/json/monster_special_attacks/spells.json

```
@@ -215,5 +215,69 @@
     "max_aoe": 3,
     "effect": "summon",
     "effect_str": "mon_centipede_small"
+  },
+  {
+    "type": "SPELL",
+    "id": "small_raptor_spawn",
+    "name": { "str": "Spawn Raptors" },
+    "description": "Summons several zombiespawn raptors.",
+    "flags": [ "HOSTILE_SUMMON", "RANDOM_DAMAGE", "PERMANENT" ],
+    "valid_targets": [ "ground", "hostile" ],
+    "min_damage": 3,
+    "max_damage": 4,
+    "min_range": 3,
+    "max_range": 3,
+    "min_aoe": 2,
+    "max_aoe": 2,
+    "base_casting_time": 80,
+    "extra_effects": [ { "id": "spawning_zombie_die", "hit_self": true } ],
+    "effect": "summon",
+    "effect_str": "mon_spawn_raptor"
+  },
+  {
+    "type": "SPELL",
+    "id": "big_raptor_spawn",
+    "name": { "str": "Spawn Raptors" },
+    "description": "Summons many zombiespawn raptors.",
+    "flags": [ "HOSTILE_SUMMON", "RANDOM_DAMAGE", "PERMANENT" ],
+    "valid_targets": [ "ground", "hostile" ],
+    "min_damage": 2,
+    "max_damage": 5,
+    "min_range": 3,
+    "max_range": 3,
+    "min_aoe": 2,
+    "max_aoe": 3,
+    "base_casting_time": 80,
+    "extra_effects": [ { "id": "spawning_zombie_die", "hit_self": true } ],
+    "effect": "summon",
+    "effect_str": "mon_spawn_raptor"
+  },
+  {
+    "type": "SPELL",
+    "id": "shady_raptor_spawn",
+    "name": { "str": "Spawn Shady Raptors" },
+    "description": "Summons a few shady zombiespawn raptors.",
+    "flags": [ "HOSTILE_SUMMON", "RANDOM_DAMAGE", "PERMANENT" ],
+    "valid_targets": [ "ground", "self" ],
+    "min_damage": 1,
+    "max_damage": 2,
+    "min_aoe": 2,
+    "max_aoe": 2,
+    "base_casting_time": 500,
+    "extra_effects": [ { "id": "spawning_zombie_die", "hit_self": true } ],
+    "effect": "summon",
+    "effect_str": "mon_spawn_raptor_shady"
+  },
+  {
+    "type": "SPELL",
+    "id": "spawning_zombie_die",
+    "name": { "str": "Spawning Zombie Die" },
+    "description": "For zombies that are spawning something and die in the process.",
+    "flags": [ "LOUD", "RANDOM_DAMAGE" ],
```

Random damage isn't needed when it's 250 for min and max.
```suggestion
    "flags": [ "NO_EXPLOSION_VFX", "SILENT" ],
```